lib/src/lints/avoid_using_debug_print/avoid_using_debug_print_rule.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/syntactic_entity.dart';
+import 'package:analyzer/error/listener.dart';
+import 'package:custom_lint_builder/custom_lint_builder.dart';
+import 'package:solid_lints/src/lints/avoid_using_debug_print/models/avoid_using_debug_print_func_model.dart';
+import 'package:solid_lints/src/models/rule_config.dart';
+import 'package:solid_lints/src/models/solid_lint_rule.dart';
+
+/// Gives warnings about using debugPrint
+///
+/// ### Example:
+/// ```dart
+/// debugPrint(info)
+/// ```
```

Comment:
Please follow the convention for how we describe lints. Example: https://github.com/solid-software/solid_lints/blob/master/lib/src/lints/no_empty_block/no_empty_block_rule.dart

Reply:
I'd probably do:
BAD: debugPrint
GOOD: logging harness of your choice